modules/op-installing-tkn-on-linux.adoc

```diff
@@ -0,0 +1,54 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/tkn_cli/installing-tkn.adoc
+
+[id="installing-tkn-on-linux"]
+
+= Installing {op-title} on Linux
+
+== Binary installation
+
+. Download the binary:
++
+----
+# curl -L https://mirror.openshift.com/pub/openshift-v4/clients/tkn/latest/tkn-linux-amd64 -o /usr/local/bin/tkn
```

Comment:
@nikhil-thomas : Could you please provide me the links for the tar.gz and binary installation files for tkn. This is applicable for all the installation links in  all the 3 files :
- op-installing-tkn-on-linux.adoc
- op-installing-tkn-on-macos.adoc
- op-installing-tkn-on-windows.adoc 

Thanks!!

Reply:
@nkakkar81: Most probably we will ship the `tar.gz` insted of direct binary. We will confirm it shortly and update it here.

cc: @vdemeester @piyush-garg @nikhil-thomas